src/index/update.rs

```diff
@@ -322,8 +341,8 @@ pub fn populate(db: &DB) -> Result<()> {
 		Regex::new(&settings.index_album_art_pattern)?
 	};
 
-	let (directory_sender, directory_receiver) = channel();
-	let (song_sender, song_receiver) = channel();
+	let (directory_sender, directory_receiver) = crossbeam_channel::unbounded();
```

Comment:
Switched to `crossbeam-channel` because it's `Sender` is `Sync` (and it's also faster and more reliable than the `std` one).